code/controllers/subsystem/garbage.dm

```diff
@@ -342,92 +342,110 @@ SUBSYSTEM_DEF(garbage)
 
 /// Qdel Item: Holds statistics on each type that passes thru qdel
 /datum/qdel_item
-	var/name = "" //!Holds the type as a string for this type
-	var/qdels = 0 //!Total number of times it's passed thru qdel.
-	var/destroy_time = 0 //!Total amount of milliseconds spent processing this type's Destroy()
-	var/failures = 0 //!Times it was queued for soft deletion but failed to soft delete.
-	var/hard_deletes = 0 //!Different from failures because it also includes QDEL_HINT_HARDDEL deletions
-	var/hard_delete_time = 0 //!Total amount of milliseconds spent hard deleting this type.
-	var/hard_delete_max = 0 //!Highest time spent hard_deleting this in ms.
-	var/hard_deletes_over_threshold = 0 //!Number of times hard deletes took longer than the configured threshold
-	var/no_respect_force = 0 //!Number of times it's not respected force=TRUE
-	var/no_hint = 0 //!Number of times it's not even bother to give a qdel hint
-	var/slept_destroy = 0 //!Number of times it's slept in its destroy
-	var/qdel_flags = 0 //!Flags related to this type's trip thru qdel.
-	var/list/extra_details //!Lazylist of string metadata about the deleted objects
+	/// Holds the type as a string for this type
+	var/name = ""
+	/// Total number of times it's passed thru qdel.
+	var/qdels = 0
```

Comment:
do you just prefer this or someodd?

Reply:
[AUTODOC](https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md#documenting-a-variabledefine)

Reply:
https://github.com/SpaceManiac/SpacemanDMM/tree/master/crates/dmdoc#documenting-code

Reply:
I wasn't getting the comment on hover :/

Reply:
we should fix that this is accepted syntax

Reply:
I'm fine with //! but it does need to be in the readme if we plan on using it

Reply:
//! seems bugged at the moment and does not offer tooltips 
![Code_jFoJMJBNXL](https://github.com/tgstation/tgstation/assets/42397676/7673f261-6c44-4f71-8f1f-21a46a2eff74)

Reply:
odd, it's supposed to work

Reply:
if it's broken we should fix it rather then removing uses of it

Reply:
I'll revert these comments and make an issue report

Reply:
o7